network/stations.csv

```diff
@@ -2126,6 +2126,7 @@ WKOK,XX,Wilberforce Kokatahi,-42.89413,171.11023,62,,WGS84,2015-01-08T00:00:00Z,
 WKTO,XX,Waikouaiti,-45.60496,170.70383,85,,WGS84,2014-09-17T00:00:00Z,9999-01-01T00:00:00Z
 WKZ,NZ,Wanaka,-44.827021353,169.017561979,558,,WGS84,2004-06-03T00:00:00Z,9999-01-01T00:00:00Z
 WL,NZ,Wellington Hogben's Kandhallah House,-41.233,174.783,360,,Wiki,1899-01-01T00:00:00Z,1916-01-01T00:00:00Z
+WLCZ,SM,Wellington College,-41.304416,174.782848,47,,WGS84,2024-10-16T03:30:00Z,9999-01-01T00:00:00Z
```

Comment:
elevation in the ticket is `38m` instead of `47`?

Reply:
The elevation I got from the GPS on the Centaur said 38 at the time. Sits now at 42 (after aqcuiring more satelites) 
Google earth elevation is 47... The truth is proable someshwere in the the middle 🙂

Reply:
We'll go with an elevation of 42m then?